src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -4540,7 +4908,7 @@ void CalcZoneMassBalance(EnergyPlusData &state, bool const FirstHVACIteration)
             }
 
             Real64 FinalTotalReturnMassFlow = 0;
-            CalcZoneReturnFlows(state, ZoneNum, StdTotalReturnMassFlow, FinalTotalReturnMassFlow);
+            zoneEquipConfig.calcReturnFlows(state, StdTotalReturnMassFlow, FinalTotalReturnMassFlow);
```

Comment:
Moved to a method so this can be shared by `zoneEquipConfig` and `spaceEquipConfig`.